src/Utilities/Blaze.hpp

```diff
@@ -18,15 +21,9 @@
 
 // Override padding, streaming and kernel options
 #define _BLAZE_SYSTEM_OPTIMIZATIONS_H_
-namespace blaze {
-constexpr bool usePadding = false;
-constexpr bool useStreaming = true;
-constexpr bool useOptimizedKernels = true;
-}
 
-namespace blaze {
-constexpr bool useDefaultInitialization = false;
-}
+const blaze::AlignmentFlag blaze_unaligned = blaze::AlignmentFlag::unaligned;
+const blaze::PaddingFlag blaze_unpadded = blaze::PaddingFlag::unpadded;
 
```

Comment:
Do you think it would make sense to add a type alias for the `blaze::Group<0>` that you use in a few places? Then you could add some comments what the group is used for.